crates/bevy_window/src/lib.rs

```diff
@@ -99,8 +100,7 @@ impl Plugin for WindowPlugin {
         if settings.add_primary_window {
             let window_descriptor = app
                 .world
-                .get_resource::<WindowDescriptor>()
-                .cloned()
+                .remove_resource::<WindowDescriptor>()
```

Comment:
This might be breaking if somehow some users rely on that `WindowDescriptor` resource. Though I don't see a usecase where that is needed.

Reply:
Using `WindowDescriptor` as a source of information about the window is also a bit of a footgun.
I'll add a migration guide :+1: